graphics/vpype/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vpype
+version             1.9.0
+
+checksums           rmd160  b56bed854d574f69bf3c264d5c08c7061e238f6d \
+                    sha256  1b09fcd67ba911d8f558a09e3da6ae655ed921b7de599ebcacc7090550f77a71 \
+                    size    476328
+
+description         The Swiss-Army-knife command-line tool for plotter vector graphics.
+long_description    ${description}
+
+categories          graphics
+platforms           darwin
+supported_archs     noarch
+maintainers         @abey79 openmaintainer
+license             MIT
+
+homepage            https://github.com/abey79/vpype
+
+python.default_version 39
```

Comment:
MacPort's current default Python version is 3.10 - can we do the same for `vpype`?

Reply:
Officially, vpype 1.9 doesn't support 3.10 yet, but I'm working on it for vpype 1.10 (abey79/vpype#417). The issues are mostly due to dependencies, so by further patching setup.py I might be able to make it work if this is important.

Reply:
Actually, this would require to update both py-multiprocess and py-pyside2 for compatibility with 3.10. I guess this is beyond the scope of this PR.